doc/source/timedeltas.rst

```diff
@@ -363,6 +363,12 @@ or ``np.timedelta64`` objects. Passing ``np.nan/pd.NaT/nat`` will represent miss
    pd.TimedeltaIndex(['1 days', '1 days, 00:00:05',
                      np.timedelta64(2,'D'), datetime.timedelta(days=2,seconds=2)])
 
+   In [5]: pd.TimedeltaIndex(['0 days', '10 days', '20 days'])
+   Out[5]: TimedeltaIndex(['0 days', '10 days', '20 days'], dtype='timedelta64[ns]', freq=None)
+
+   In [4]: pd.TimedeltaIndex(['0 days', '10 days', '20 days'], freq='infer')
```

Comment:
Can you put this in a `.. ipython:: python` block?  A new block will need to be created if you add the sentence I suggested above.  This essentially allows sphinx to generate the python output when the docs are being built.  You'll only need to add the input line (without `In [4]:`) since the output will be generated.  See the existing code sample a few lines above for an example of how to set this up.